CHANGELOG.md

```diff
@@ -0,0 +1,62 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+
+## [Unreleased]
+
+### Added
+
+* Integrating APIs with security from [@ohltyler](https://github.com/ohltyler)
+
+### Changed 
+* Corrected ml/stats/nodes permissions spelling from [@camAtGitHub](https://github.com/camAtGitHub)
+* Internal user usage clarification from [@reshippie](https://github.com/reshippie)
+* Remove `advanced_modules_enabled` examples from [@wjordan](https://github.com/wjordan)
+* CHANGELOG.md replaces release-notes
+
+### Security
+
+* Security patch for `org.cryptacular:cryptacular`
```

Comment:
if this is about version updates i'd suggest to list the actual versions to which you updated, this makes it easier to glance at the changelog to see whether the version is acceptable (or has other issues by the time somebody wants to update to this release).

e.g.:
```suggestion
* Update to `org.cryptacular:cryptacular:1.2.4` (includes security fixes for [...])
```
note: i briefly checked the history and #1806 which bumped this version doesn't say _what_ the security issue was. this is IMHO bad practice, it should list the known CVEs or at least point to _somewhere_ where they're listed.

Reply:
Including the relevant CVEs and version numbers is a great callout.